actioncable/lib/rails/generators/channel/templates/assets/channel.js

```diff
@@ -0,0 +1,21 @@
+(function() {
```

Comment:
We don't need it actually. I think I had copied the wrapper from `cable.js` file. I removed it now.